contrib/dex/README.md

```diff
@@ -0,0 +1,65 @@
+## Exposing DEX service
+After the cluster is up, you have to manually expose dex service using a ELB or Ingress.
```

Comment:
@camilb Not necessary in this PR but just my two cents.

I guess it will be even better if we could enable users to create an elb and a record set configured appropriately by writing cluster.yaml like we do for apiEndpoints[]:

```yaml
dex:
  exposedBy:
    loadBalancer:
      hostedZone:
        id: hostedzone-abcdefg
```

so that we can ensure people set up dex in a more automated, fail-proof way.